CompanyManager.DAL/Entities/Post.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CompanyManager.DAL.Entities
+{
+    public class Post
```

Comment:
Ta klasa jest OK i zostaje, później jak ją dodamy do PersonContextu to jakieś drobne zmiany pewnie będą potrzebne. Na razie OK, tak tylko informacyjnie podaję.